modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/MostPopularItemsList.ts

```diff
@@ -0,0 +1,30 @@
+module app.create {
+
+    export class MostPopularItemsList extends NewContentDialogList {
+
+        constructor() {
+            super();
+        }
+
+        createItemView(item: MostPopularItem): api.dom.LiEl {
```

Comment:
createItemView method dublicates the same from parent class - NewContentDialogList. It could be removed.